glustercli/cmd/rebalance.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/gluster/glusterd2/pkg/restclient"
+	rebalance "github.com/gluster/glusterd2/plugins/rebalance/api"
+)
+
+const (
+	helpRebalanceCmd       = "Gluster Rebalance"
+	helpRebalanceStartCmd  = "Start rebalance session for gluster volume"
+	helpRebalanceStatusCmd = "Status of rebalance seesion"
+	helpRebalanceStopCmd   = "Stop rebalance session"
+)
+
+var (
+	flagRebalanceStartCmdForce     bool
+	flagRebalanceStartCmdFixLayout bool
+)
+
+var rebalanceCmd = &cobra.Command{
+	Use:   "volume rebalance",
+	Short: helpRebalanceCmd,
+}
+
+func init() {
+
+	// Rebalance Start
+	rebalanceStartCmd.Flags().BoolVar(&flagRebalanceStartCmdForce, "force", false, "Force")
```

Comment:
The description of the flags can be a little bit more elaborate.

Reply:
For now I am following the trend set by volumestart, will update both of them with more description in the next PR